app/api/orders.py

```diff
@@ -395,20 +395,19 @@ def before_get_object(self, view_kwargs):
         """
         if view_kwargs.get('attendee_id'):
             attendee = safe_query(
-                self, TicketHolder, 'id', view_kwargs['attendee_id'], 'attendee_id'
+                TicketHolder, 'id', view_kwargs['attendee_id'], 'attendee_id'
             )
             view_kwargs['id'] = attendee.order.id
         if view_kwargs.get('order_identifier'):
             order = safe_query(
-                self,
                 Order,
                 'identifier',
                 view_kwargs['order_identifier'],
                 'order_identifier',
             )
             view_kwargs['id'] = order.id
         elif view_kwargs.get('id'):
-            order = safe_query(self, Order, 'id', view_kwargs['id'], 'id')
+            order = safe_query(Order, 'id', view_kwargs['id'], 'id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)